src/lib/sanity.queries.ts

```diff
@@ -38,14 +38,11 @@ export interface Post {
   description?: string
   longDescription?: string
   readMe?: boolean
-  size?: number
-  mainImage?: ImageAsset
   releaseNotes: PortableTextBlock[]
   releaseDate: PortableTextBlock[]
```

Comment:
What about releaseDate ?

Reply:
release date i was going to leave in because eventually we will use it, we just don't have the data yet. Shall i just get rid of it?

Reply:
In that case no keep it